plugins/woocommerce/templates/checkout/form-coupon.php

```diff
@@ -12,7 +12,7 @@
  *
  * @see https://woocommerce.com/document/template-structure/
  * @package WooCommerce\Templates
- * @version 7.0.1
+ * @version 9.7.0
```

Comment:
```suggestion
 * @version 9.8.0
```

I noticed that the CI expects the version of this template file to be bumped to `9.8.0`, see https://github.com/woocommerce/woocommerce/actions/runs/13068377368/job/36464594617?pr=54577.